queries.sql

```diff
@@ -15,3 +15,53 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Transaction-1 */
+BEGIN;
+UPDATE animals SET species = 'Unspecified';
+ROLLBACK;
+
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 

>Inside a transaction update the animals' table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

- [ ] According to the project requirement, after setting the species column to `unspecified`, you need to verify that the changes were made and also include a query for that. Also after you roll back the changes, you are meant to verify that everything went back to the initial state. In that more, I'll suggest you update this query to include the two verifications. here's what it should look like. :+1: :+1: 

```sh
   BEGIN;
   UPDATE animals
   SET species = 'unspecified';
   SELECT species from animals; 
   ROLLBACK;
   SELECT species from animals;
```